### PR TITLE
add mirror for Prometheus

### PR DESCRIPTION
**Pull request by mjudeikis**
add prometheus img promotion
cc @openshift/sig-azure

### CODE REVIEW

File: projects/azure/image-mirror/image-mirror.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ items:
   data:
     openshift-azure_v3_10_quay: |
       registry.svc.ci.openshift.org/azure/azure-plugins:sync quay.io/openshift-on-azure/sync:v3.10 quay.io/openshift-on-azure/sync:v3.10.0 quay.io/openshift-on-azure/sync:latest
+      registry.svc.ci.openshift.org/azure/azure-misc:prometheus-sidecar quay.io/openshift-on-azure/prometheus-sidecar:v3.10 quay.io/openshift-on-azure/prometheus-sidecar:v3.10.0 quay.io/openshift-on-azure/prometheus-sidecar:latest
```
Comment (kargakis) [inline projects/azure/image-mirror/image-mirror.yaml]: Do you have a PR somewhere for adding the azure-misc imagestream?

Comment (jim-minter) [reply projects/azure/image-mirror/image-mirror.yaml]: why would we have a prometheus-sidecar:v3.10 ?

Comment (mjudeikis) [reply projects/azure/image-mirror/image-mirror.yaml]: it just matches versioning of all other. latest only?

**Comment by kargakis** [comment]
If it's created by the ci-operator then we probably don't need it. But we
should ensure that is the case.

**Comment by mjudeikis** [comment]
> If it's created by the ci-operator then we probably don't need it. But we
should ensure that is the case. 

I never created them, and they are there. We can add the just to be safe.

**Comment by mjudeikis** [comment]
ping @kargakis

**Pull request merged**